### PR TITLE
Added missing event handler removal methods for C interop.

### PR DESCRIPTION
**Pull request by jeuxjeux20**
Those methods were missing for some reason, now they're there!

### CODE REVIEW

File: include/NovelRT.Interop/NrtNovelRunner.h [hunk 1]
```diff
@@ -28,10 +28,18 @@ extern "C"
     NrtResult Nrt_NovelRunner_getRenderer(NrtNovelRunnerHandle runner, NrtRenderingServiceHandle* outputService);
     NrtResult Nrt_NovelRunner_getDebugService(NrtNovelRunnerHandle runner, NrtDebugServiceHandle* outputService);
 
-    NrtResult Nrt_NovelRunner_addUpdate(NrtNovelRunnerHandle runner, void (*func)(NrtTimestamp, void*), void* context);
+    NrtResult Nrt_NovelRunner_addUpdate(NrtNovelRunnerHandle runner,
```
Comment (RubyNova) [inline include/NovelRT.Interop/NrtNovelRunner.h:31]: where is the remove counterpart to this?

**Review by RubyNova — COMMENTED** [review]
LGTM but I do have a concern - also I believe we should swap out add/remove for subscribe/unsubscribe or some variant thereof. :)

File: include/NovelRT.Interop/NrtNovelRunner.h (hunk 1 above)
Comment (jeuxjeux20) [reply include/NovelRT.Interop/NrtNovelRunner.h:31]: Line 35

Comment (RubyNova) [reply include/NovelRT.Interop/NrtNovelRunner.h:31]: damn I'm blind. Cool.

**Review by RubyNova — APPROVED** [review]

**Pull request merged**